cogboard-webapp/src/components/Board/styled.js

```diff
@@ -19,10 +19,16 @@ export const StyledNotFound = styled(NotFound)`
   position: absolute;
   right: 0;
   top: 0;
-  z-index: 2000;
 
   > div {
     padding: 0 15px;
     text-align: center;
   }
 `;
+
+export const StyledNotFound = styled(BaseNoData(NotFound))`
+  background: #211f39;
+  z-index: 2000;
```

Comment:
Is this ok ? is board drawer still accessible?

Reply:
Drawer is accessible when there are no boards and app is showing component responsible for add first board (scope of this pull request). Drawer won't be accessible when app is showing NotFound component, but it wasn't showing drawer before this PR as well.

Reply:
Can You send me a screenshot on `Slack`